docs/serviceability.md

```diff
@@ -0,0 +1,58 @@
+# Serviceability
+
+## Logs
+
+### Deployed Locally
+View the application-service controller logs by running the manager executable. Example,
+
+```
+make install
+make build
+./bin/manager
+```
+
+### Deployed on a Cluster
+View the application-service controller logs by tailing the manager container log of the controller manager pod. Example,
+
+```
+oc logs -f application-service-application-service-controller-manager -c manager
+```
+
+### Understanding the Logs
+Each application-service controller logs their reconcile logic to the manager. The log message format is generally of format 
+
+```
+2023-03-14T08:39:25.009Z	INFO	controllers.Component	updating devfile component name kubernetes-deploy ...	{"appstudio-component": "HAS", "Component": "mynamespace/pipeline-devfile-example-lcec", "clusterName": ""}
+```
+
+The above log indicates that the message is logged by the `Component` controller for the resource `pipeline-devfile-example-lcec` in the namespace `mynamespace`.
+
+## Debugging
+
+- Insert break points at the controller functions to debug unit tests
+- When debugging unit and integration tests, remember that the mock clients used are hosting dummy data and mocking the api call and returns 
+
+## Common Problems
+- A Github Personal Access token is required as the application-service controller requires the token for pushing the resources to the GitOps repository
```

Comment:
Are you suggesting users use their own github PAT to create gitops repos under their own user org? if so, link to these instructions https://github.com/redhat-appstudio/application-service#creating-a-github-secret-for-has.  We should also follow github's suggestion to use a fine grained token instead of the classic one: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#fine-grained-personal-access-tokens

Reply:
Updated